main_spec.tex

```diff
@@ -135,6 +135,18 @@ \subsection{Communication Management Routines}
 default context.  The default context can be used explicitly through the
 \LibHandleRef{SHMEM\_CTX\_DEFAULT} handle.
 
+\newtext{
+Every communication context has an associated \ac{PE} team.
+This \ac{PE} team specifies the set of \acp{PE} over which \ac{PE}-specific
```

Comment:
I have been thinking that it should be very clear that assigning a team to a context changes the PE numbering in routines that take a context argument. This statement makes it clear that the context means that we are operating over a specific set, which should also imply that the numbering is relative to the set, but maybe it is worth explicitly stating that the "team specifies the set of PEs and the numbering of those PEs over which..."

Reply:
:+1: (I expected that we'd want to expand the context-team interaction text a bit more. Thanks for the suggestion.)